db/src/lib.rs

```diff
@@ -1,6 +1,27 @@
 pub use sqlx;
 use sqlx::postgres::PgPoolOptions;
 pub use sqlx::PgPool;
+use color_eyre::Result;
+
+
+#[derive(sqlx::FromRow)]
+pub struct User {
+    pub user_id: String,
```

Comment:
Oh I think I told you to use a `String` here, so sorry I'm about to tell you to change it 😆 

But I think we can use the `Uuid` type from the `uuid` crate here!

That way its clearer from just the Rust code that we are storing a `uuid` here as opposed to just any arbitrary string!